azurelinuxagent/common/logger.py

```diff
@@ -155,7 +155,7 @@ def __init__(self, level, event_func):
         self.event_func = event_func
 
     def write(self, level, msg):
-        if self.level <= level:
+        if self.level >= level:
```

Comment:
Why was this reversed? What was it doing before?

Reply:
Good catch. Fixed it and added a test to capture it.